themes/classic/_dev/package.json

```diff
@@ -10,26 +10,27 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "autoprefixer": "^6.7.7",
     "babel-loader": "^5.3.2",
-    "bootstrap": "4.0.0-alpha.4",
+    "bootstrap": "4.0.0-alpha.5",
```

Comment:
Because Alpha 6 switch to complete flexbox and we would need to rework the grid. We will do that when the first beta version will be out.